src/test/java/com/example/produktapi/model/ProductTest.java

```diff
@@ -65,10 +65,37 @@ void checkingImageDetailFor_product1() {
 
     @Test //LinnBergstroem
     void changePriceProduct1() {
-        System.out.println(product1.getPrice());
         product1.setPrice((double) 20.0);
         Assertions.assertEquals(20, product1.getPrice());
-        System.out.println(product1.getPrice());
+    }
+
+    // Tests by Swapnal Hardikar
+    @Test
+    void setTitle_shouldSetTitle() {
+        String expectedTitle = "New Title";
+        String oldTitle = product1.getTitle();
+        product1.setTitle(expectedTitle);
+        Assertions.assertEquals(expectedTitle,product1.getTitle());
+        Assertions.assertNotEquals(oldTitle,product1.getTitle());
```

Comment:
Generellt bra kod, men du måste använda `Assertions.assertAll()` om du ska köra flera assertions. Se kodförslaget nedan.

```suggestion
        Assertions.assertAll(
                () -> assertEquals(expectedTitle,product1.getTitle()),
                () -> assertNotEquals(oldTitle,product1.getTitle())
        );
```